be/src/exec/pipeline/set/except_build_sink_operator.cpp

```diff
@@ -26,8 +26,7 @@ Status ExceptBuildSinkOperator::close(RuntimeState* state) {
 Status ExceptBuildSinkOperatorFactory::prepare(RuntimeState* state) {
     RETURN_IF_ERROR(OperatorFactory::prepare(state));
 
-    RowDescriptor row_desc;
-    Expr::prepare(_dst_exprs, state, row_desc);
+    Expr::prepare(_dst_exprs, state, _row_desc);
```

Comment:
You needn't change it. we will remove RowDescriptor from Expr::prepare soon.

Reply:
it's already changed, _row_desc now in OperatorFactory.